Bottom: 9cf068be2a5c14a44d9a2169039b29342a6d0852
Top:    4d9830e07d29a23344d4c7337f1011e3d1cc90f0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-05-15 18:09:26 +0100

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 3747233..1c755f7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
